packages/meditrak-server/src/tests/routes/importSurveys/assertCanImportSurveys.test.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import { AccessPolicy } from '@tupaia/access-policy';
+import { buildAndInsertSurveys } from '@tupaia/database';
+import { TUPAIA_ADMIN_PANEL_PERMISSION_GROUP } from '../../../permissions';
+import { getModels } from '../../getModels';
+import { assertCanImportSurveys } from '../../../routes/importSurveys/assertCanImportSurveys';
+
+const DEFAULT_POLICY = {
+  DL: ['Public'],
+  KI: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+  SB: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Royal Australasian College of Surgeons'],
+  VU: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+  LA: ['Admin'],
+};
+
+describe('assertCanImportSurveys(): Permissions checker for Importing Surveys', async () => {
+  const models = getModels();
+
+  before(async () => {
+    const adminPermissionGroup = await models.permissionGroup.findOne({ name: 'Admin' });
+    const vanuatuCountry = await models.country.findOne({ code: 'VU' });
+
+    await buildAndInsertSurveys(models, [
+      {
+        code: 'TEST_SURVEY',
+        name: 'Test Survey',
+        permission_group_id: adminPermissionGroup.id,
+        country_ids: [vanuatuCountry.id],
+      },
+    ]);
+  });
+
+  describe('Sufficient permissions when importing existing surveys', async () => {
```

Comment:
All other files are structured as:
```
describe('Some action') {
  it('Sufficient permissions')
  it('Insufficient permissions')
}
```

But this one is structured as:
```
describe('Sufficient permissions') {
  it('Some action')
  it('Some action')
}
```

Did we maybe wanna shuffle this around to keep it consistent?

Reply:
@EMcQ-BES Sorry for getting back late. I think for `assertCanImportSurveys.test.js`, there's a big difference in scenarios/test cases between importing new surveys and importing existing surveys. But for others (eg: `assertCanImportEntities.test.js`) there are none. So that's why I thought particularly for this one, it's better to break them down more so that the structure is more readable

Eg:

- assertCanImportSurveys.test.js
```
describe('assertCanImportSurveys(): Permissions checker for Importing Surveys')
	describe('Sufficient permissions when importing existing surveys')
		testCase()
		testCase()
		testCase()
	describe('Sufficient permissions when importing new surveys')
		testCase()
		testCase()
		testCase()
	describe('Insufficient permissions when importing existing surveys')
		testCase()
		testCase()
		testCase()
	describe('Insufficient permissions when importing new surveys')
		testCase()
		testCase()
		testCase()
```


- assertCanImportEntities.test.js
```
describe('assertCanImportEntities(): Permissions checker for Importing Entities')
	testCase()
	testCase()
	testCase()
```

Reply:
I think that's a reasonable justification